src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseRepository.java

```diff
@@ -183,6 +184,14 @@ public Claim linkLetterHolder(Long claimId, String letterHolderId) {
         return coreCaseDataService.linkLetterHolder(claimId, letterHolderId);
     }
 
+    @Override
+    public Claim updateClaimSubmissionOperationStatus(String authorisation, Long claimId,
```

Comment:
paramertes need to be arranged

Reply:
We have to agree. arranging one parameter per line seems not really needed. If its standard, then it needs to be questioned